packages/yoastseo/src/parsedPaper/ParsedPaper.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * A parsed version of the paper, containing a tree representation of the text,
+ * and parsed metaData pertaining to the tree's context.
+ *
+ * @class ParsedPaper
+ *
+ * @memberOf module:parsedPaper
+ * */
+export default class ParsedPaper {
+	/**
+	 *
```

Comment:
You could add a simple description, like
```suggestion
	 * Creates a new ParsedPaper.
```